telegram/message.py

```diff
@@ -2986,6 +3000,8 @@ def text_markdown(self) -> str:
             :attr:`telegram.ParseMode.MARKDOWN` is is a legacy mode, retained by Telegram for
             backward compatibility. You should use :meth:`text_markdown_v2` instead.
 
+            |custom_emoji_formatting_note|
+
```

Comment:
Note:

Reply:
no its in the upper block

Reply:
ah sorry, hadn't expanded the diff … but can we make it an itemized list, then?

Reply:
You just want me to work more grrr.

Will do.